admin-dev/themes/new-theme/js/app/pages/stock/components/overview/spinner.vue

```diff
@@ -97,6 +98,11 @@
         this.value = '';
         this.product.qty = null;
       },
+      onKeydown(event: KeyboardEvent): void {
```

Comment:
Can you add a comment there to explain why you added this? It will be clearer if we need to change it in the future